applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -441,6 +473,18 @@ static void broadcast_config_print(const struct shell *shell,
 			}
 		}
 
+		const unsigned char *program_info;
+
+		ret = bt_audio_codec_cfg_meta_get_program_info(codec_cfg, &program_info);
+		if (ret > 0) {
+			shell_print(shell, "\t\tProgram info: %s", program_info);
+		}
+
+		int immediate =
```

Comment:
bt_audio_codec_cfg_meta_get_bcast_audio_immediate_rend_flag() might return -EINVAL, we would need to handle that.